.eslintrc.js

```diff
@@ -32,6 +35,29 @@ module.exports = {
     jest: true,
   },
   settings: {
+    'import/order': [
+      'error',
+      {
+        pathGroups: [
+          {
+            pattern: '~/**',
+            group: 'external',
+            position: 'after',
+          },
+        ],
+        groups: [
+          'builtin',
```

Comment:
what do these groups do?

Reply:
They define how we want the imports to sort. 
More [info here](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md)